js/output/pjs/pjs-code-injector.js

```diff
@@ -156,13 +156,46 @@ class PJSCodeInjector {
 
             playSound: (sound) => {
                 if (sound && sound.audio && sound.audio.play) {
+					if( sound.audio.fadeOutInterval ) {
+						sound.audio.fadeOutStopInterval();                                    
+					}
+
                     sound.audio.currentTime = 0;
                     sound.audio.play();
                 } else {
                     throw {message: i18n._("No sound file provided.")};
                 }
             },
 
+			stopSound: (sound, fadeOut) => {
+				if (sound && sound.audio && sound.audio.volume) {
+					if( !fadeOut || fadeOut < 0.020 ) {
+						sound.audio.pause();
+					} else {
+						var volume = sound.audio.volume,
+							orgVolume = volume;
```

Comment:
Can you make these two separate `var` declarations?  

Also, what if `stopSound` gets called during fadeout?  It might be safer to set `orgVolume` to whatever the default volume for `<audio>` tags seeing as there's no processing API to set the volume.